src/constants/physics/CODATA.hpp

```diff
@@ -0,0 +1,44 @@
+#ifndef NJOY_CONSTANTS_PHYSICS_CODATA_HPP
```

Comment:
Why the tag structs in a CODATA header. Please move rename  this file

Reply:
I'm not sure what you mean here. Why shouldn't the header guard be in this file?

Reply:
This file isn't specific to CODATA, but it is named `CODATA.hpp`. 

Suggested alternative: `tags.hpp`

In addition, this file is not specific to a constants library (e.g. CODATA) and doesn't benefit from separate inclusion as implied by the header guard. Prefer top down inclusion for files other than the library files (as opposed to the definition files, e.g. `CODATA2014.hpp`)